### PR TITLE
Clean up wording in Action Mailer Basics [ci skip]

### PR DESCRIPTION
**Pull request by scottgonzalez**
Just added some missing words. The two halves of the sentence are now worded the same.

### CODE REVIEW

File: guides/source/action_mailer_basics.md [hunk 1]
```diff
@@ -160,8 +160,8 @@ When you call the `mail` method now, Action Mailer will detect the two templates
 #### Calling the Mailer
 
 Mailers are really just another way to render a view. Instead of rendering a
-view and sending out the HTTP protocol, they are just sending it out through the
-email protocols instead. Due to this, it makes sense to just have your
+view and sending it out through the HTTP protocol, they are just sending it out
```
Comment (vipulnsward) [inline guides/source/action_mailer_basics.md]: how about using "sending it over the HTTP protocol"

Comment (scottgonzalez) [inline guides/source/action_mailer_basics.md]: updated

**Comment by vipulnsward** [comment]
@scottgonzalez thanks, can you squash the commits.

**Comment by scottgonzalez** [comment]
squashed

**Comment by vipulnsward** [comment]
Thanks!

**Pull request merged**